### ISSUE TITLE
More than >1'000'000 Iterations

### ISSUE THREAD

**Issue by Masi69** [xaos-project/XaoS#249]
Hello, is it possible to run these fractals with more than 1'000'000 iterations? Thank you very much!

**Comment by martinwguy** [comment]
Yes. With 1,000,000 iterations I've zoomed in on the spike between the first two bulbs until it showed no more of the promontories, then set the iterations to 10,000,000 and it promptly showed more of the fractal. What makes you think you can't?
